app/models/procedure.rb

```diff
@@ -289,13 +297,24 @@ def switch_list_order(list, index_of_first_element)
   end
 
   def clone(admin, from_library)
+    # FIXUP: needed during transition to revisions
+    # Ther find / reload is needed to not break attachment clone
+    RevisionsMigration.add_revisions(Procedure.find(id))
```

Comment:
j'ai l'impression qu'on a déjà la `procedure`, du coup ça doit pouvoir s'alléger un poil non ?:
```suggestion
    RevisionsMigration.add_revisions(self)
```

Reply:
Alors, idéalement oui. Mais il y a un truc chelou qui se passe avec le clonage des attachements. Et c'est le fix. Je n’ai pas testé avec rails 6. Je vais le faire.

Reply:
trop bien, avec rails 6 ça passe sans le hack !